src/comp/comp.py

```diff
@@ -23,49 +27,54 @@ def __repr__(self):
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with 'D':
+name = humans[0].name
+print(len(name))
+print(name[0])
 print("Starts with D:")
-a = []
+a = [person.name for person in humans if person.name[0] == "D"]
 print(a)
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name ends in "e".
 print("Ends with e:")
-b = []
+b = [person.name for person in humans if person.name[len(
+    person.name)-1] == "e"]
```

Comment:
To make this more readable I would stick to how you did the first example like:
`b = [h.name for h in humans if h.name[-1:] == "e"]`